src/Hosting/ServiceCollectionExtensions.cs

```diff
@@ -17,13 +18,15 @@ public static class ServiceCollectionExtensions
 		/// <summary>
 		/// Add Omex Logging and ActivitySource dependencies
 		/// </summary>
+		[Obsolete($"Please do not use {nameof(AddOmexServices)} if you just want to use Legacy OmexLogger and ActivityEventSender because they are deprecated. Consider using a different telemetry solution. This method is pending for removal by 1 July 2024.", DiagnosticId = "OMEX188")]
 		public static IHostBuilder AddOmexServices(this IHostBuilder builder) =>
 			builder
 				.ConfigureServices((context, collection) => collection.AddOmexServices());
 
 		/// <summary>
 		/// Add Omex Logging and ActivitySource dependencies
 		/// </summary>
+		[Obsolete($"Please do not use {nameof(AddOmexServices)} if you just want to use Legacy OmexLogger and ActivityEventSender because they are deprecated. Consider using a different telemetry solution. This method is pending for removal by 1 July 2024.", DiagnosticId = "OMEX188")]
```

Comment:
Is OmexActivitySource getting deprecated?

Reply:
This source is deprecated [EventSource(Name = "Microsoft-OMEX-TimedScopes-Ext")]

Reply:
I'm asking about ActivitySource registered from .AddOmexActivitySource(). It looks like it will stay with some changes, so you can't obsolete this method.

Reply:
oh okay. I use Pragma instead